messages/rest.md

```diff
@@ -0,0 +1,33 @@
+# summary
+
+Make an authenticated HTTP request to Salesforce REST API and print the response.
+
+# examples
```

Comment:
I think you should do a PATCH example (ex: edit a field on account from stdin) and a POST example (ex: create a record from a file)